data/json/items/generic.json

```diff
@@ -311,18 +311,22 @@
     "flags": [ "TRADER_AVOID", "FIREWOOD", "TINDER" ]
   },
   {
-    "type": "GENERIC",
+    "type": "AMMO",
     "id": "superglue",
     "category": "spare_parts",
     "symbol": ",",
+    "phase": "liquid",
     "color": "white",
     "name": { "str_sp": "superglue" },
     "description": "A tube of strong glue.  Used in many crafting recipes.",
     "price": "18 USD",
     "price_postapoc": "1 USD",
     "material": [ "plastic" ],
-    "weight": "8 g",
-    "volume": "10 ml",
+    "weight": "1 g",
+    "volume": "1 ml",
+    "ammo_type": "superglue",
+    "//": "1 charge of superglue should cover an area of 0.5mm^2",
+    "count": 40,
```

Comment:
You do realize that with this, a single 'charge' of superglue is 0.025ml, right?

Reply:
Shit, meant 5mm^2